docs/troubleshooting/large-partition-table.rst

```diff
@@ -111,9 +113,10 @@ Large partitions are stored in a system table with the following schema:
        sstable_name text,
        partition_size bigint,
        partition_key text,
-       rows bigint,
-       range_tombstones bigint,
        compaction_time timestamp,
+       dead_rows bigint,
+       range_tombstones bigint,
+       rows bigint,
```

Comment:
here too, lets describe the actual schema.

Reply:
Hmm, I wonder if this isn't a bug with DESCRIBE TABLE.
It should produce a compatible schema definition, and that should have exactly the same order of columns.
Maybe I'm missing something.
@tgrabiec / @denesb any insights?

Reply:
When schema is persisted to schema tables, the order in which columns were passed to the schema builder is lost. Order of regular columns after reading from schema tables is lexicographical by column name. So "describe" output is correct.

Reply:
Yes, column matching happens by name, not by index in the schema object. You can read old sstables using newer schema which has more columns.

Reply:
For some reason I remembered we're relying on the column id, not its name, maybe for column rename...